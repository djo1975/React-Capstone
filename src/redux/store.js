import countrySlice from './countries/countrySlice';

const { configureStore } = require('@reduxjs/toolkit');

const store = configureStore({
  reducer: {
    countries: countrySlice,
  },
});

export default store;
