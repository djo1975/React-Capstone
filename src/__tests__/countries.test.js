import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import store from '../redux/store';
import Countries from '../components/countries';

describe('', () => {
  it('testing render component', () => {
    const count = renderer.create(
      <Provider store={store}>
        <BrowserRouter>
          <Countries />
        </BrowserRouter>
      </Provider>,
    ).toJSON();
    expect(count).toMatchSnapshot();
  });
});
