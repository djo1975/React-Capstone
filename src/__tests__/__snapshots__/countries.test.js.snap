// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` testing render component 1`] = `
<div>
   
  <div
    className="dropdown"
  >
    <img
      alt="Globe"
      src="pngfind.com-continents-png-2212530 (1).png"
    />
    <h1
      className="title"
    >
      World
    </h1>
    <select
      id="continents"
      name="continents"
      onChange={[Function]}
    >
      <option
        value="all"
      >
        All coutries
      </option>
      <option
        value="Africa"
      >
        Africa
      </option>
      <option
        value="Antarctica"
      >
        Antarctica
      </option>
      <option
        value="Asia"
      >
        Asia
      </option>
      <option
        value="Europe"
      >
        Europe
      </option>
      <option
        value="North America"
      >
        North America
      </option>
      <option
        value="Oceania"
      >
        Oceania
      </option>
      <option
        value="South America"
      >
        South America
      </option>
    </select>
  </div>
  <div
    className="dragisa"
  />
</div>
`;
